cypress/support/reusableConstants.ts

```diff
@@ -1,6 +1,7 @@
 export const planWithChargesName = `a plan with charges`
+export const planWithChargeCode = `a_plan_with_chargesnew_code_plan_with_charge`
```

Comment:
Any chance it's a wrong paste in the string and the value should be shorter?

Seems we pasted next to an existing value trying to replace it, imo 😬

Reply:
nice catch